st4/FileDecoder.py

```diff
@@ -0,0 +1,59 @@
+from LSP.plugin.core.protocol import Error
+from LSP.plugin.core.registry import LspTextCommand
+from LSP.plugin.core.typing import Any
+from LSP.plugin.core.url import filename_to_uri  # TODO: deprecated in a future version
+
+import os
+import sublime
+
+class FileDecoderCommand(LspTextCommand):
+    session_name = "metals"
+
+    _jvm_extentions = {'java', 'scala', 'class'}
+    _tasty_extentions = {'scala', 'tasty'}
+    _scala_extentions = {'scala', 'tasty'}
+    _sematicdb_extentions = {'scala', 'java', 'semanticdb'}
+    _decoders = {
```

Comment:
Can you double check this is correct ?
I got it from here: https://github.com/scalameta/metals/blob/3f211b03d6c2d1ff4d5fa936d50a4aab013100fb/metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala#L185

Reply:
Looks good!